diff --git a/pallet-balances/Cargo.toml b/pallet-balances/Cargo.toml
index eb44b3af..7b0b8a90 100644
--- a/pallet-balances/Cargo.toml
+++ b/pallet-balances/Cargo.toml
@@ -193,3 +193,11 @@ priority = 2
 [lints.rust.suspicious_double_ref_op]
 level = "allow"
 priority = 2
+
+[dependencies.sp-io]
+version = '37.0.0'
+default-features = false
+
+[dependencies.either]
+version = "1.8.1"
+default-features = false
diff --git a/pallet-indices/Cargo.toml b/pallet-indices/Cargo.toml
index ed482f98..07011b75 100644
--- a/pallet-indices/Cargo.toml
+++ b/pallet-indices/Cargo.toml
@@ -196,3 +196,7 @@ priority = 2
 [lints.rust.suspicious_double_ref_op]
 level = "allow"
 priority = 2
+
+[dependencies.log]
+version = "0.4.21"
+default-features = false
